content/root/quick-start-guide.textile

```diff
@@ -93,20 +93,32 @@ blang[php].
   bc[php]. require_once __DIR__ . '/../vendor/autoload.php';
 
 blang[java].
-  The Ably Client Library SDK is hosted on "GitHub":https://github.com/ably/ably-java.
+  The Ably Client Library SDK is available on "GitHub":https://github.com/ably/ably-java and hosted on "Maven Central":https://mvnrepository.com/repos/central.
 
-  To get started with your project, add the SDK into your @build.gradle@ dependencies section.
+  To get started with your project, add the SDK into your @build.gradle@ dependencies section and a reference to Maven in the repositories section if it isn't included by default.
 
   For Java applications:
 
   ```[java]
-  compile 'io.ably:ably-java:1.2.0'
+  dependencies {
+    implementation 'io.ably:ably-java:1.2.0'
+  }
+
+  repositories {
+  	mavenCentral()
+  }
   ```
 
   For Android apps:
 
   ```[java]
-  compile 'io.ably:ably-android:1.2.0'
+  dependencies {
+    implementation 'io.ably:ably-android:1.2.0'
+  }
+
+  repositories {
+  	mavenCentral()
```

Comment:
The indentation needs fixing in the `repositories` section.

Reply:
This was odd, as both `dependencies` and `repositories` were displaying as a single tab in my IDE. I've replaced the `repositories` with double spaces and checked in web view and that fixes it.